app/(auth)/signup/page.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from 'next/link';
+
+import SignUpForm from './_components/signupForm';
+
+export default function SignUp() {
+  return (
+    <div className="flex flex-col items-center">
+      <SignUpForm />
+      <p className="mt-5 text-black">
+        이미 가입하셨나요?{' '}
+        <span className="text-[#5534DA] underline">
+          <Link href="/signin">로그인하기</Link>
```

Comment:
여기서도 상수화해서 다뤄주면 좋을 것 같아요!
```suggestion
          <Link href={ROUTE_PATHS.SIGNIN}>로그인하기</Link>
```